tests/test_settings.py

```
@@ -1,53 +1,36 @@
+
 from pathlib import Path
 import os
-from endoreg_db.utils.paths import STORAGE_DIR
-from endoreg_db.logger_conf import get_logging_config # Import the function
+
+from base_settings import (
+    INSTALLED_APPS,
```

Ruff (0.11.9)

`base_settings.INSTALLED_APPS` imported but unused

Remove unused import

(F401)